velodrome/fetcher/README.md

```diff
@@ -1,3 +1,58 @@
+Overview
+========
+
+Fetcher retrives a github repository history and stores it in a MySQL local
+database.
+
+For now, it downloads three types of resources:
+- Issues (including pull-requests)
+- Events (associated to issues)
+- Comments (regular comments and review comments)
+
+All of these resources will allow us to:
+- Compute average time-to-resolution for an issue/pull-request
+- Compute time between label creation/removal: lgtm'd, merged
+- break-down based on specific flags (size, priority, ...)
+
+The model is described more precisely in [../sql](../sql/).
+
+Fetcher only downloads what is not already in the SQL database by trying to find
+the latest events it knows about. It will poll from Github on a regular basis,
+and this can be configured with the `--frequency` flag.
+
+There is no set-up required as `fetcher` will create the github database if it
+doesn't exist, along with the various required tables.
+
+Design decision
+===============
+
+Because of the size of the kubernetes/kubernetes repository:
+
+- Accessing comments API doesn't work and returns `500 Internal Error` all the
```

Comment:
This one:

https://api.github.com/repos/kubernetes/kubernetes/issues/comments

I talked about that with an engineer from Github, looks like they fixed it now, great! We can definitely improve the code now :-)